packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -57,10 +62,43 @@ class TokenDisabledError extends Error {
     this.name = 'TokenDisabledError'
   }
 }
+
+function getDefaultTokenName(address: string) {
+  const lowercased = address.toLowerCase()
+  return (
+    lowercased.substring(0, 5) +
+    '...' +
+    lowercased.substring(lowercased.length - 3)
+  )
+}
+
+function getDefaultTokenSymbol(address: string) {
+  const lowercased = address.toLowerCase()
+  return (
+    lowercased.substring(0, 5) +
+    '...' +
+    lowercased.substring(lowercased.length - 3)
+  )
+}
+
+export interface TokenBridgeParams {
+  l1: {
+    signer: Signer
+    network: L1Network
+  }
+  l2: {
+    signer: Signer
+    network: L2Network
+  }
+}
+
 export const useArbTokenBridge = (
   bridge: Bridge,
-  autoLoadCache = true
+  autoLoadCache = true,
```

Comment:
let's put the param with a default fallback last (I thought the static checker enforced this 🤔 )